platform-tests/src/platform/acceptance/tls_version_test.go

```diff
@@ -0,0 +1,91 @@
+package acceptance_test
+
+import (
+	"crypto/tls"
+	"net/url"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Checking the TLS version", func() {
+
+	testHostPortGetters := map[string]func() string{
+		"apps_apex": func() string {
+			return testConfig.GetAppsDomain() + ":443"
```

Comment:
This block is running outside an `It` or a `Before*`, and therefore is executed before the `BeforeSuite` has run. I would therefore expect `testConfig` to be nil at this point.  Additionally `Expect` calls can't be used in this context - if they fail, they cause a runtime panic because there's nothing in place to handle them.

Reply:
Actually, scratch that. Now that I read this more closely I understand what's going on, and this is returning a function that is then executed in the relevant context.

Reply:
?

Reply:
You crossed your text. Answering anyway, it is a map of getters, functions that return a string. 

The reason is that the values of `testConfig` are set in the `BeforeSuite` as you mention, so I need to delay the evaluation. Each test call the getter to get the right value.